Makefile.env

```diff
@@ -0,0 +1,2 @@
+export STORAGE_ONEDRIVE_INTEGRATION_TEST=on
+export STORAGE_ONEDRIVE_CREDENTIAL=file:<abs_path_of_credential>
```

Comment:
Remove `Makefile.env` from the repo, since in the README it says using `cp Makefile.env.example Makefile.env` to create this file. And add `-include Makefile.env` to `Makefile` so that this file can be used.